template.yaml

```diff
@@ -37,6 +37,33 @@ Resources:
         CertificateArn: !Ref AcmCertificateArn
         EndpointConfiguration: EDGE
 
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      OwnershipControls:
+        Rules:
+          - ObjectOwnership: ObjectWriter
+      PublicAccessBlockConfiguration:
+        BlockPublicPolicy: False
+      AccessControl: PublicRead
```

Comment:
AWS recommends disabling ACLs therefore you don't need this.

take a look at the [s3 docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html) for `Granting public access to S3 buckets`

Reply:
thanks, I was trying to follow that doc but got confused when AWS::Include complained about access to the object. it turns out I wasn't correctly setting the `ListObjects` permission on the bucket in the policy